packages/firestore/test/integration/api/transactions.test.ts

```diff
@@ -510,13 +510,12 @@ apiDescribe('Database transactions', persistence => {
   });
 
   asyncIt('cannot have a get without mutations', () => {
-    return integrationHelpers.withTestDb(persistence, db => {
-      const docRef = db.doc('foo/bar');
```

Comment:
In this case, I would just leave the old code (so you don't have to to docRef.firestore later). You can still use .doc() to ensure randomness.

Reply:
Okay. Not sure it makes much difference, but switched back in the spirit of not changing things unnecessarily. :-)